linode/terraform-provider-linode
# new: Add `linode_placement_group` data source

## 📝 Description

Adding the data source `linode_placement_group` to read detailed information about a Linode placement group.

## ✔️ How to Test

```
make PKG_NAME="linode/placementgroup" int-test
```

**Manual test:**

1. In a sandbox environment, i.e. dx-devenv, run the following script to read the linode_placement_group data source.
```tf
data "linode_placement_group" "test" {
    id = {your-pg-id}
}

output "result" {
    value = data.linode_placement_group.test
}
```
2. Observe the detailed information of the above placement group is outputed.
